backend/main-service/src/main/java/kkakka/mainservice/order/ui/OrderController.java

```diff
@@ -1,9 +1,38 @@
 package kkakka.mainservice.order.ui;
 
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import kkakka.mainservice.member.application.MemberService;
+import kkakka.mainservice.order.application.OrderService;
+import kkakka.mainservice.product.application.ProductService;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 @RestController
 @RequestMapping("/order")
```

Comment:
```suggestion
@RequestMapping("/orders")
```

REST 표현은 복수가 더 낫겠네요!

Reply:
네 수정하겠습니다~